fast-sim-and-analysis/FccFastSimAnalysis.md

```diff
@@ -23,71 +23,64 @@ git clone https://github.com/HEP-FCC/FCCAnalyses.git
 cd FCCAnalyses
 ```
 
-This tutorial runs with fcc-edm so we need to checkout the corresponding branch
-
-
-```bash
-git checkout fccedm
-```
-
 and follow the compilation instructions [here](https://github.com/HEP-FCC/FCCAnalyses/#getting-started) to get started with **ONLY** the installation.
```

Comment:
in principle, FCCAnalyses is installed on cvmfs, so this should work without compiling anything, but maybe you also want people to know how it is done?

Reply:
this is true, for this tutorial where no code development is needed, we could remove the local installation. While a more advanced tutorial could be written to add a new algorithm. I can do it later. And BTW, in master a performance issue was fixed, I just made a new release, could we deploy it? I'd prefer if this tutorial depends on it
https://github.com/HEP-FCC/FCCAnalyses/releases/tag/v0.3.2